### DISCUSSION TITLE
Messages error

### DISCUSSION THREAD

**Discussion by liexes** [diamond-rip/Eden#313] [Q&A]
I wanted to bring to your attention an issue that I have been experiencing with the plugin. The plugin messages are displaying errors and instead of the actual text, I'm seeing messages like "title.winner". I would appreciate your help in resolving this issue.

**Comment by GoodestEnglish** [comment]
May I have a picture and see what is actually happening?

**Comment by liexes** [comment]
Sorry, i fix it
